packages/help-center/src/components/utils.tsx

```diff
@@ -14,3 +15,19 @@ export const generateContactOnClickEvent = (
 		} );
 	}
 };
+
+export const calculateUnread = ( conversations: ZendeskConversation[] ) => {
```

Comment:
Maybe we can use this utility also in https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/components/help-center-chat-history.tsx#L39-L41 the chat history?

Reply:
done